### DISCUSSION TITLE
Missed 'Copy to clipboard' icon in Code Block when using content.code.copy and navigation.instant

### DISCUSSION THREAD

**Discussion by syntezzz1** [squidfunk/mkdocs-material#5500] [General]
### Context

https://github.com/squidfunk/mkdocs-material/discussions/5471

### Bug description

When updating mkdocs-material from 8.5.11 to 9.x.x ([according to this instruction](https://squidfunk.github.io/mkdocs-material/upgrade/#upgrading-from-8x-to-9x)) the button for copying code blocks is missing. I added the `content.code.copy` feature, but the buttons never appeared. I tried to figure out which feature this feature might conflict with and realized that the features `content.code.copy` and `navigation.instant` do not work at the same time. When `navigation.instant` is disabled, copy buttons appear.

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
- [Upgrading from 8.x to 9.x](https://squidfunk.github.io/mkdocs-material/upgrade/#upgrading-from-8x-to-9x)
- [Code copy button](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#code-copy-button)
- [Instant loading](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/?h=navigation+instant#instant-loading)

### Reproduction

[codeblock_issue.zip](https://github.com/squidfunk/mkdocs-material/files/11438715/codeblock_issue.zip)

### Steps to reproduce

1. Unpack the archive and go to the directory `codeblock_issue`
```
uzip codeblock_issue.zip 
cd codeblock_issue
```
2. Run the command:
```
mike deploy --update-aliases v.1.1.1
```
3. Make sure that in the file `site/Index.html` there are buttons for copying code blocks
4. Open the file `mkdocs.yml` and uncomment the line `- navigation.instant`
5. Run the command:
```
mike deploy --update-aliases v.1.1.1
```
6. Make sure that in the file `site/Index.html` there are no buttons for copying code blocks

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting. Does it work locally when running `mkdocs serve`?

**Comment by syntezzz1** [comment]
> Thanks for reporting. Does it work locally when running `mkdocs serve`?

Sorry, forget about `--update-aliases`.

Yes, it works with `mkdocs serve`

**Comment by squidfunk** [comment]
Then it is very likely related to one of the other plugins. The example you provided is far from being minimal – there are 3 third-party plugins involved: `img2fig`, `awesome-pages` and `mike`. Please further reduce the example so that only the necessary plugins and extensions are present, i.e., it is minimal, because the error could be induced by any of them. With that many dependencies, we're not able to help you.

Also look in the browser console and check if there are errors. If there are, please post them here. If you have your site deployed, please post a link to your site. Thank you.

**Comment by syntezzz1** [comment]
Sorry, I double-checked again. With `mkdocs serve` both features work. The error is caused precisely when executed via `mike deploy --update-aliases v.1.1.1`. Is mike probably the problem?

I have removed all these plugins, everything remains the same. There are no errors in the logs when executing `mike deploy --update-aliases v.1.1.1`.
Here is the archive without plugins:
[codeblock_issue.zip](https://github.com/squidfunk/mkdocs-material/files/11438975/codeblock_issue.zip)

**Comment by squidfunk** [comment]
There are errors in the console in your 9.1.8 deployment, complaining about missing files. Those errors do not appear on your 8.5.11 deployment. Please fix those and check whether the issue persists:

<img width="1439" alt="Bildschirm­foto 2023-05-10 um 09 22 42" src="https://github.com/squidfunk/mkdocs-material/assets/932156/1cc1479f-7657-45e0-85c2-80bd9e361c55">

Converting to discussion, since this doesn't look like an issue with Material for MkDocs.

**Reply by syntezzz1** [reply]
Yes, I fixed them and now the problem doesn't persist. Thank you very much.
